docs/source/endpoints/relations.md

```diff
@@ -0,0 +1,246 @@
+---
+myst:
+  html_meta:
+    'description': 'Create, query, and delete relations between content items with the /@relations endpoint.'
```

Comment:
I'm not sure how special characters will affect SEO, so I would just drop them from the HTML meta tags.

```suggestion
    'description': 'Create, query, and delete relations between content items with the relations endpoint.'
```